docs/pages/access-graph.mdx

```diff
@@ -82,6 +82,68 @@ resources based on their role or team.
 Resources are the things that users can or cannot access. They can be
 servers, databases, or Kubernetes clusters.
 
+## Searching
+
+To search for a node, either click the search button in the sidebar on the left,
```

Comment:
I think that 'a node' is a bit confusing in the context of Teleport. My first thought was an SSH node. I'd use "to search in the graph" or something similar. For that reason, I tried to avoid the word "node" when writing the first version.

@ptgott Thoughts on this?

Reply:
@ryanclark @jakule I think "graph node" would work here, since we've already mentioned this as a heading. To make this clearer, we could also say, "To search for a graph node, such as a user or an action...".